docs/source/user/file_formats.md

```diff
@@ -1,162 +1,151 @@
-
 # File and output formats
 
-## Overview
-
-When working with code and data, you will encounter data and files in a wide
-variety of formats. JupyterLab provides a unified architecture for working with
-data. This model applies whether the data is in a file, or is provided by a
-Kernel as output in a notebook or code console.
+JupyterLab provides a unified architecture for viewing and editing data in a
+wide variety of formats. This model applies whether the data is in a file or is
+provided by a kernel as rich cell output in a notebook or code console.
 
 For files, the data format is detected by the extension of the file. A single
 file extension may have multiple editors or viewers registered. For example a
-Markdown file (`.md`) can be edited in the File Editor, or rendered inline.
-
-You can access different editors and viewers for a file, by right-clicking on
-the filename in the File Browser and using the "Open With..." submenu:
+Markdown file (`.md`) can be edited in the file editor or rendered and displayed
+as HTML. You can open different editors and viewers for a file by right-clicking
+on the filename in the file browser and using the "Open With..." submenu:
 
 [screenshot]
 
-To use these different data formats as output in the Notebook or Code Console,
-you can use the relevant display API for the kernel you are using. For example,
-the IPython kernel provides a `display` function that takes a `dict` of keys
-(MIME types) and value (MIME data):
+To use these different data formats as output in a notebook or code console, you
+can use the relevant display API for the kernel you are using. For example, the
+IPython kernel provides a variety of convenience classes for displaying rich output:
 
 ```python
-from IPython.display import display
-display({'text/html': '<h1>Hello World</h1>'}, raw=True)
+from IPython.display import display, HTML
+display(HTML('<h1>Hello World</h1>'))
 ```
 
-Running this code will display the HTML in the output of the notebook or code
-console:
+Running this code will display the HTML in the output of a notebook or code
+console cell:
 
 [screenshot]
 
-Other Jupyter Kernels offer similar APIs.
+The IPython display function can also construct a raw rich output message from a
+dictionary of keys (MIME types) and values (MIME data):
```

Comment:
Do we want to provide docs links to outside pages in the IPython or Notebook docs?

Reply:
This comment is the same as @ian-r-rose

Reply:
actually, I'm not sure the best way to have stable links. Can we punt this to a new issue?